### DISCUSSION TITLE
text input

### DISCUSSION THREAD

**Discussion by SiNiN-me** [PetrIvan/chord-seq-ai-app#45] [Feature requests]
First of all, thank you so much for making this app. It's huge for me. Instead of drawing every MIDI note by hand just to try out some ideas, this app helps me so much. Patreon is on my to-do list now, LOL!

The only feature I need to request now is:

Can you add text input? Something like:

Typing 'Cmaj7 / Am7 / Dm7 / G7'
Pressing enter
This would help a lot when using ChatGPT to jumpstart ideas or something similar.

❤️

**Comment by PetrIvan** [comment]
Hi, thank you for the feedback!

I could include it as another import/export option for text files, although I am not sure how to handle direct text input. Dedicating space for a relatively niche feature isn't optimal, Ctrl + V is unintuitive, and showing an overlay after pressing import would slow down other import file types that are more frequent.

Another issue is that there isn't a universal way to parse chords from their notation algorithmically, but there could be some fallback when a chord isn't recognized.

In the meantime, you can search for each chord and add it one by one. Let me know if you have any suggestions!
